test/drip/client/configuration_test.rb

```diff
@@ -109,4 +109,17 @@ class Drip::Client::ConfigurationTest < Drip::TestCase
       assert_equal 42, config.http_timeout
     end
   end
+
+  context "#skip_analytics" do
+    should "accept passed parameter" do
+      config = Drip::Client::Configuration.new(skip_analytics: true)
+      assert_equal true, config.skip_analytics
```

Comment:
Optional, but I think using `assert` instead of `assert_equal` will let you omit the `true` and just directly check that `config.skip_analytics` is `true`

Reply:
Honestly, though, I'm not sure if I prefer that over this more explicit approach :P

Reply:
@BenSandeen I honestly just copied another test and didn't even think much of it. I do think the explicit nature of it is good since you would set this as an option if you wanted to use it.

Reply:
When these bool assertions fail, the message will be something like: "expected true to be false", which isn't always helpful. You can add a msg string argument to `assert` for context, which can be helpful, ie:

```
assert config.skip_analytics, "config should skip analytics"
```